Consolify snapshot documentation

abeyad opened the pull request:
This commit fixes the snapshot documentation to be in conformity
with the CONSOLE format.

nik9000 commented on docs/reference/modules/snapshots.asciidoc:
```diff
@@ -24,7 +24,7 @@ Elasticsearch. The repository settings are repository-type specific. See below f
 [source,js]
 -----------------------------------
-PUT /_snapshot/my_backup
+PUT _snapshot/my_backup
```
The leading `/` is fine.

nik9000 replied:
No need to remove it.

nik9000 commented on docs/reference/modules/snapshots.asciidoc:
```diff
@@ -24,7 +24,7 @@ Elasticsearch. The repository settings are repository-type specific. See below f
+PUT _snapshot/my_backup
 {
   "type": "fs",
   "settings": {
```
Maybe this should have an example instead of `...`?

nik9000 replied:
And this can be `// CONSOLE` too.

nik9000 replied:
You could do `// CONSOLE` and `// TESTESTUP` and that way the repository is created for every snippet.

nik9000 commented on docs/reference/modules/snapshots.asciidoc:
```diff
@@ -62,23 +62,26 @@ can be obtained using the following command:
-GET /_snapshot/_all
+GET _snapshot/_all
 -----------------------------------
+// CONSOLE
```
Do these pass? Like, I expect most of them to 404 unless you do something to make sure the repository exists.

abeyad commented:
@nik9000 I pushed https://github.com/elastic/elasticsearch/pull/23189/commits/d14a71639e8de8503d19eb9b9a783c743ac41e80 which fixes the snapshot doc so they work with the docs tests.  I also addressed your feedback and changed the snippets to have a leading `/`.
